templates/_components/_image.twig

```diff
@@ -63,9 +63,9 @@
 	Preset options:
 	---------------
 	@option   media          {string}      Any valid media query
-	@option   breakpoints    {array}       Array of transform values for different view breakpoints
-	@option   sizes          {array}       Array of size based transform values
-	@option   transform      {array}       Array of aspect and position related transform values
+	@option   breakpoints    {array}       Transform values for different view breakpoints
+	@option   sizes          {array}       Transform values for size that will render as the `sizes` attribute in the `<img>` tag
```

Comment:
This is incorrect. The `guide` param of presets is rendered as the `sizes` attribute of the `<img>` (or could even be `<source>`) tag. The `sizes` parameter is _not_ rendered as the `sizes` attribute.

Reply:
Fixed.